Worktez/worktez
# fix(TeamsCard):Added Member Profile Picture

### Functionality:

- Fixed Duplicate Teams Issue.
-Replaced Members' profile picture instead of Emails

### Solution:

- Fetched the Team Details Instead of checking the Existing Team Detail.
- Created a profile-picture component

### Risk level:

### How to test:

- Go to My organization check whether there are duplicate teams.
- Add Members and check whether the profile pictures are displayed.
